src/module/entities/helper.js

```diff
@@ -1885,11 +1891,17 @@ module.exports = class UserProjectsHelper {
 						})
 
 						if (!updateData['metaInformation.name'] || !updateData['metaInformation.externalId']) {
-							singleEntity.status = CONSTANTS.apiResponses.ENTITIES_FAILED
+							singleEntity.status = CONSTANTS.apiResponses.ENTITIES_UPDATE_FAILED
 							singleEntity.message = CONSTANTS.apiResponses.FIELD_MISSING
 							return singleEntity
 						}
-						// let data =
+						console.log(updateData, 'line no 1788')
+
+						if (/^[0-9a-fA-F]{24}$/.test(updateData['metaInformation.externalId'])) {
```

Comment:
@MallanagoudaB why can't we check 
const { ObjectId } = require("mongodb");

if (ObjectId.isValid(updateData.metaInformation.externalId)) {

please check which approach is better. regex or this